src/map.rs

```diff
@@ -1487,27 +1490,33 @@ where
     /// ]);
     /// assert_eq!(
     ///     got,
-    ///     Some([
-    ///         &mut 1807,
-    ///         &mut 1800,
-    ///     ]),
+    ///     [
```

Comment:
I think it would help if the docs showed an example using the let..else syntax to extract the results:
```rust
let [Some(a), Some(b)] = map.get_many_mut(..) else { panic!() };
```

Reply:
Added let..else examples.